README.md

```diff
@@ -23,15 +25,96 @@ Require in `Capfile` to use the default task:
 require 'capistrano/nvm'
 ```
 
-Configurable options:
+The task then uses command maps to run `nvm.sh` when necessary, as controlled by the setting `:nvm_map_bins`.
+
+## Configuration
+
+### Settings
+
+In your capistrano `config/deploy.rb`:
+
+#### Required
+
+- `:nvm_node`
+  - Accepts: String
+  - Default: None
+  - Notes: The version number must be prefixed with a `v` if `:nvm_verb` is set to `:use`. See examples below.
+
+#### Optional
+
+- `:nvm_type`
+  - Accepts: `:user`, `:system`
+  - Default: `:user`
+  - Notes: Used to determine the location of `nvm.sh`, similar to setting `$NVM_HOME` in your shell.
+
+- `:nvm_custom_path`
+  - Accepts: String
+  - Default: Unused
+  - Notes: Setting this effectively overrides `:nvm_type`.
+
+- `:nvm_map_bins`
+  - Accepts: Ruby word array
+  - Default: `%w{node npm yarn}`
+  - Notes: This is a list of commands that will be executed with NVM support by capistrano.
```

Comment:
The nvm abbreviation is supposed to written in lowercase, Capistrano on the other hand should be capitalized.

Reply:
Fixed this and the other occurrences of improper capitalization